compel/arch/arm/plugins/std/syscalls/syscall.def

```diff
@@ -22,6 +22,7 @@ rt_sigprocmask	sigprocmask	135	175	(int how, k_rtsigset_t *set, k_rtsigset_t *ol
 rt_sigreturn			139	173	(void)
 ioctl				29	54	(unsigned int fd, unsigned int cmd, unsigned long arg)
 pread64				67	180	(unsigned int fd, char *buf, size_t count, loff_t pos)
+pread				67	180	(unsigned int fd, char *buf, size_t count, loff_t pos)
```

Comment:
There's no `pread()` syscall on aarch64/arm32.
Aliasing one to another shouldn't be done with a hack in syscall table file, rather in a C header.
Besides, what's the reason why you just use `pread64()` instead of `pread()`?

Reply:
Sure, I know there's no `pread()` on aarch64. However, these syscall functions are autogenerated when compiling and related to system architecture(for example, in my ubuntu 20.04 virtual machine, there is no sys_pread64() syscall). So I think the only solution is to add this into syscall table.

Reply:
Ok, I see what you mean, the other platforms are already "renaming" it in their syscall tables:
```
compel/arch/mips/plugins/std/syscalls/syscall_64.tbl:__NR_pread64                       5016            sys_pread
compel/arch/ppc64/plugins/std/syscalls/syscall-ppc64.tbl:__NR_pread64           179             sys_pread
compel/arch/s390/plugins/std/syscalls/syscall-s390.tbl:__NR_pread64             180             sys_pread
compel/arch/x86/plugins/std/syscalls/syscall_32.tbl:__NR_pread64                180             sys_pread64
compel/arch/x86/plugins/std/syscalls/syscall_64.tbl:__NR_pread64                        17              sys_pread
```
Could you than just "rename" it too `pread64 = > pread`, rather than adding a new alias?

Reply:
OK, considering that there's no `sys_pread64()` used in this project.